### PR TITLE
tweak to use external verifier

### PR DESCRIPTION
**Pull request by ohpaul**
- uses `Verifier.sol` from `seal-cred-verifier-contract`

### CODE REVIEW

File: contracts/SCERC721Derivative.sol [hunk 1]
```diff
@@ -59,4 +59,8 @@ contract SCERC721Derivative is ERC721, Ownable {
   {
     return super.supportsInterface(_interfaceId);
   }
+
+  function setVerifierAddress(address _verifier) external onlyOwner {
+    verifier = _verifier;
```
Comment (backmeupplz) [inline contracts/SCERC721Derivative.sol:64]: should we make `verifier` public?

File: scripts/deploy.ts [hunk 2]
```diff
@@ -14,7 +14,8 @@ async function main() {
   } as { [chainId: number]: string }
   const chainName = chains[chainId]
   const SealCred = await ethers.getContractFactory('SealCredLedger')
-  const sealCred = await SealCred.deploy()
+  const verifierAddress = '0x0000000000000000000'
```
Comment (backmeupplz) [inline scripts/deploy.ts:17]: Let's prompt user

**Review by backmeupplz — CHANGES_REQUESTED** [review]

File: contracts/SCERC721Derivative.sol (hunk 1 above)
Comment (ohpaul) [reply contracts/SCERC721Derivative.sol:64]: The `verifier` variable is `private` for this file but the `setVerifierAddress` function is `external` and `onlyOwner`, should this be an internal function then?

File: .env.example [hunk 3]
```diff
@@ -1,3 +1,3 @@
 ETHERSCAN_API_KEY=0000000000000000000000000000000000000000
-ETH_RPC=https://example.com/rpc
+RPC_URL=https://eth.example.com/rpc
```
Comment (backmeupplz) [inline .env.example:2]: No need for this change

**Review by backmeupplz — APPROVED** [review]

**Pull request merged**